app/models/card.rb

```diff
@@ -1,4 +1,7 @@
+require 'elasticsearch/model'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.